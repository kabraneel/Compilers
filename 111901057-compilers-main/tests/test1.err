IR PRETTY PRINT
.globl main
main:
li t1, 1
li t8, 10
syscall

 BASIC BLOCKS 
.globl main

main:
li $t1, 1
li $v0, 10
syscall

