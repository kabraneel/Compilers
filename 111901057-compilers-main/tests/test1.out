.globl main
main:
li $t1, 1
li $v0, 10
syscall
